src/plugins/data/public/ui/saved_query_flyouts/open_saved_query_flyout.tsx

```diff
@@ -301,25 +343,12 @@ export function OpenSavedQueryFlyout({
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiButton
-              disabled={!selectedQuery}
+              // disabled={!selectedQuery}
```

Comment:
not blocking, but can we clean up dead code?

Reply:
Fixed